src/patternfly/components/Masthead/masthead-toggle.hbs

```diff
@@ -0,0 +1,16 @@
+<span class="pf-c-masthead__toggle{{#if masthead-toggle--modifier}} {{masthead-toggle--modifier}}{{/if}}"
+  {{#if masthead-toggle--attribute}}
+    {{{masthead-toggle--attribute}}}
+  {{/if}}>
+  {{#if @partial-block}}
+    {{> @partial-block}}
+  {{else}}
+    {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Masthead toggle"'}}
```

Comment:
Technically this will need `aria-expanded`, too. I expect it might be fair for someone to come here and copy/paste this code into their project? I wonder if the `aria-label` should be "Global navigation", too.